README.md

```diff
@@ -141,6 +141,18 @@ user_item_similarity = DerivedFeature(name="user_item_similarity",
                                       transform="cosine_similarity(user_embedding, item_embedding)")
 ```
 
+## Cloud Integrations
+
+| Feathr component             | Cloud Integrations                                                                            |
+| ---------------------------- | --------------------------------------------------------------------------------------------- |
+| Offline store – Object Store | Azure Blob Storage, Azure ADLS Gen2, AWS S3                                                   |
+| Offline store – SQL          | Azure SQL DB, Azure Synapse Dedicated SQL Pools (formerly SQL DW), Azure SQL in VM, Snowflake |
+| Online store                 | Azure Cache for Redis                                                                         |
+| Feature Registry             | Azure Purview                                                                                 |
+| Compute Engine               | Azure Synapse Spark Pools, Databricks                                                         |
+| Machine Learning Platform    | Azure Machine Learning, Jupyter Notebook                                                      |
+| File Format                  | Parquet, ORC, Avro, Delta Lake                                                                |
```

Comment:
this is a bit hard to make sense. probably defer this to a diff rb?

Reply:
Ya I think it's like recording all the supported output format in the README file as @jaymo001 suggested, and we don't have that table for that hence also adding that table in the README file.